src/main/java/exception/CheckSplitLineImpl.java

```diff
@@ -0,0 +1,21 @@
+package exception;
+
+import java.util.Arrays;
+
+public class CheckSplitLineImpl implements CheckSplitLine {
+    private static final String SPECIAL_CHARACTER = ",";
```

Comment:
```suggestion
    private static final String COMMA = ",";
```
be more specific, any character can be special